security/security-secrets-setup.1.rst

```diff
@@ -0,0 +1,178 @@
+NAME
+====
+
+security-secrets-setup — Creates an on-device public-key infrastructure (PKI) to secure microservice secret management
+
+SYNOPSIS
+========
+
+| security-secrets-setup [-confdir <confdir>] [-p|--profile <name>] generate
```

Comment:
minor nitpick this would read better as 
```suggestion
| security-secrets-setup generate [-confdir <confdir>] [-p|--profile <name>]
```

that is easier to see the difference between the lines I think since the other options are all the same for the various subcommands

Reply:
If I accept it will invalidate the existing approvals.  Nothing is stopping you from accepting the PR and then filing your own PR to change the order of the arguments.

Reply:
if you think the other reviewers would disagree with the change that's fine, but I can re-approve and merge for you if you want to accept the suggestion, it seems like @jim-wang-intel is the only one who has already reviewed who would have their review invalidated

anyways it's a minor nitpick and I understand if you don't want to apply it, I don't feel strongly enough about it to file a PR for it though

Reply:
Changed. Please re-review.